include/kansei/fallen/node/fallen_determinant.hpp

```diff
@@ -51,11 +51,16 @@ class FallenDeterminant
 
   DeterminantType determinant_type;
 
-  // fallen raw variables
-  float fallen_back_raw_limit;
-  float fallen_front_raw_limit;
-  float fallen_right_raw_limit;
-  float fallen_left_raw_limit;
+  // fallen variables
+  float accel_back_limit;
+  float accel_front_limit;
+  float accel_right_limit;
+  float accel_left_limit;
+
+  float pitch_back_limit;
+  float pitch_front_limit;
+  float roll_right_limit;
+  float roll_left_limit;
```

Comment:
Prefer using `double` over `float`:
```suggestion
  // fallen variables
  double accel_back_limit;
  double accel_front_limit;
  double accel_right_limit;
  double accel_left_limit;

  double pitch_back_limit;
  double pitch_front_limit;
  double roll_right_limit;
  double roll_left_limit;
```

Reply:
As much as I agree, but could you still explain why, mas?

Reply:
1. We should choose to use either `float` or `double` to represent a floating-point number. Do not mix between them to prevent casting.
2. The `update_fallen_status` function uses `double` as the parameter, but inside the function, the parameter is being compared with `float` values. We should update this.
3. On modern computers, `double` is as fast as `float`, so performance doesn't matter here.